python/lsst/ts/watcher/rules/enabled.py

```diff
@@ -0,0 +1,88 @@
+# This file is part of ts_watcher.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["Enabled"]
+
+import yaml
+
+from lsst.ts import salobj
+from lsst.ts.watcher import base
+
+
+class Enabled(base.BaseRule):
+    """Monitor the summary state of a CSC.
+
```

Comment:
I think it would be good to expand the description of this rule a little bit further. Something like;

```Monitor the summary state of a CSC. Issues  `base.AlarmSeverity.SERIOUS` if CSC is in `FAULT` state and `base.AlarmSeverity.WARNING` if CSC is not ENABLED. ```

Reply:
Good idea. I have updated docs for all rules to describe the alarm severities and provide the rule name.